addons/tmf_loadouts/loadouts/ussr_vdv_1980.hpp

```diff
@@ -0,0 +1,540 @@
+		LIST_3("CUP_100Rnd_TE4_LRT4_762x54_PK_Tracer_Green_M")
+	};
+};
+
+class hmgg : car {
+	displayName = "HMG Gunner";
+	backPack[] = {"CUP_B_AGS30_Gun_Bag"};
+};
+
+class hmgac : car {
+	displayName = "HMG Ammo Carrier";
+	secondaryWeapon[] = {};
+	backPack[] = {"CUP_B_AGS30_Tripod_Bag"};
+};
+
+class hmgtl : car {
+	displayName = "HMG Team Leader";
+	backPack[] = {"CUP_B_AGS30_Tripod_Bag"};
+	items[] += {"ACE_MapTools"};
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemRadio",
+		"ItemWatch",
+		"Binocular"
+	};
+};
+
+class matg : car {
+	displayName = "MAT Gunner";
+	backpack[] = {"CUP_B_HikingPack_Civ"};
+	secondaryWeapon[] = {"CUP_launch_Metis"};
+	magazines[] += {"CUP_AT13_M"};
+};
+
+class matac : car {
+	displayName = "MAT Ammo Carrier";
+	backpack[] = {"CUP_B_HikingPack_Civ"};
+	backpackItems[] = {"CUP_AT13_M"};
+};
+
+class mattl : car {
+	displayName = "MAT Team Leader";
+	backpack[] = {"CUP_B_HikingPack_Civ"};
+	items[] += {"ACE_MapTools"};
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemRadio",
+		"ItemWatch",
+		"Binocular"
+	};
+	backpackItems[] = {"CUP_AT13_M"};
+};
+
+class hatg : car {
+	displayName = "HAT Gunner";
+	backPack[] = {"CUP_B_Metis_Gun_Bag"};
+};
+
+class hatac : car {
+	displayName = "HAT Ammo Carrier";
+	backPack[] = {"CUP_B_Metis_Tripod_Bag"};
+};
+
+class hattl : car {
+	displayName = "HAT Team Leader";
+	backPack[] = {"CUP_B_Metis_Tripod_Bag"};
+	items[] += {"ACE_MapTools"};
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemRadio",
+		"ItemWatch",
+		"Binocular"
+	};
+};
+
+class mtrg : car {
+	displayName = "Mortar Gunner";
+	secondaryWeapon[] = {"ace_csw_staticMortarCarry"};
+	backPack[] = {"CUP_B_HikingPack_Civ"};
+	backpackItems[] = {
+		LIST_4("ACE_1Rnd_82mm_Mo_HE"),
+		"ACE_1Rnd_82mm_Mo_Illum",
+		"ACE_1Rnd_82mm_Mo_Smoke"
+	};
+};
+
+class mtrac : car {
+	displayName = "Mortar Ammo Carrier";
+	backPack[] = {"CUP_B_HikingPack_Civ"};
+	backpackItems[] = {
+		LIST_4("ACE_1Rnd_82mm_Mo_HE"),
+		"ACE_1Rnd_82mm_Mo_Illum",
+		"ACE_1Rnd_82mm_Mo_Smoke"
+	};
+};
+
+class mtrtl : car {
+	displayName = "Mortar Team Leader";
+	secondaryWeapon[] = {"ace_csw_carryMortarBaseplate"};
+	items[] = { MTR_GEAR };
+	backPack[] = {"CUP_B_HikingPack_Civ"};
+	backpackItems[] = {
+		LIST_4("ACE_1Rnd_82mm_Mo_HE"),
+		"ACE_1Rnd_82mm_Mo_Illum",
+		"ACE_1Rnd_82mm_Mo_Smoke"
+	};
+
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemRadio",
+		"ItemWatch",
+		"Binocular"
+	};
+};
+
+class samg : car {
+	displayName = "AA Missile Specialist";
+	backpack[] = {"CUP_B_HikingPack_Civ"};
+	secondaryWeapon[] = {"CUP_launch_9K32Strela"};
+	magazines[] += {"CUP_Strela_2_M"};
+};
+
+class samag : car {
+	displayName = "AA Assistant Missile Specialist";
+	backpack[] = {"CUP_B_HikingPack_Civ"};
+	secondaryWeapon[] = {"CUP_launch_9K32Strela"};
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemRadio",
+		"ItemWatch",
+		"Binocular"
+	};
+	magazines[] += {"CUP_Strela_2_M"};
+};
+
+/*
+class sn : r {
```

Comment:
Do not remove roles, even if they aren't used in real life. It will otherwise throw errors in the editor if people change factions.